src/main/java/net/id/incubus_core/condition/api/package-info.java

```diff
@@ -1,2 +1 @@
-@javax.annotation.ParametersAreNonnullByDefault
```

Comment:
Por que?

Reply:
The javax lib is not included in the dependencies, making the ide not able to reference these annotations.
The lib should be added in gradle or something similar